packages/SketchMorph2-Core.package/MorphicMonet.class/instance/chooseColor.st

```diff
@@ -0,0 +1,9 @@
+tooling
+chooseColor
+	| currentColor |
+	self world activeHand showTemporaryCursor: 
+		(CursorWithAlpha fromIcon: UiFugueIcons pipetteIcon).
+	[Sensor redButtonPressed] whileFalse:
+		[currentColor := (Display colorAt: Sensor cursorPoint)].
```

Comment:
Why are we fetching the color all the time? Wouldn't it be sufficient to fetch it one time after the loop?